app/models/planning_application.rb

```diff
@@ -129,6 +129,10 @@ class WithdrawOrCancelError < RuntimeError; end
     delegate :session_id
   end
 
+  with_options on: :require_development_type do
+    validates :development_type, inclusion: [true, false]
+  end
+
```

Comment:
Since there's only one using this option can't we just do this as a one-liner?

```suggestion
  validates :development_type, inclusion: [true, false], on: :require_development_type
    
```